build.gradle

```diff
@@ -37,7 +60,7 @@ validateNebulaPom.enabled = false
 
 buildscript {
     ext {
-        opensearch_version = "2.0.0-SNAPSHOT"
+        opensearch_version = "3.0.0-SNAPSHOT"
```

Comment:
@prudhvigodithi this is right change in general, but for plugins we should target 2.0.0 at most - we expect most of the development happening here.